pkg/gotohelm/gochart.go

```diff
@@ -174,6 +199,11 @@ func (c *GoChart) render(dot *helmette.Dot) ([]kube.Object, error) {
 			return nil, errors.Newf("missing dependency %q", dep.Name)
 		}
 
+		subdot.Values, err = subchart.LoadValues(dot.Values[dep.Name])
```

Comment:
Do we need to be doing this if `LoadValues` is already recursively loading and merging values?

Reply:
You are right!
I don't know why I added this call.